Meta/gn/build/library_name.gni

```diff
@@ -0,0 +1,6 @@
+declare_args() {
+  output_name_prefix = ""
+  if (current_os != "serenity") {
+    output_name_prefix = "lagom-"
```

Comment:
Good point. I've got local changes to set it in the `alink` and `solink` tools. Not sure what to do about `solink_module`, since the llvm build doesn't seem to set the output_name in that case.